src/prompt/files_prompt.rs

```diff
@@ -9,7 +9,7 @@ use std::iter;
 pub struct FilesPrompt<'a> {
     config: &'a Config,
     checked: Vec<bool>,
-    selected_index: u16,
+    highlighted_index: u16,
```

Comment:
I changed this from `selected_...` as I felt that it did not accurately reflect the variable. I'm not sure if highlighted is the best, maybe `hovered_...` would be better?

Reply:
If you disagree I can of course change it back.

Reply:
Maybe "focused_line" or similar? At least in web stuff, a keyboard event (like pressing <kbd>space</kbd> or <kbd>d</kbd>) will target the focused element, which is what we do with this.